Assets/LeapMotionModules/ElementRenderer/Resources/LeapGui.cginc

```diff
@@ -0,0 +1,3 @@
+#include "UnityCG.cginc"
+
+#define ELEMENT_MAX 32
```

Comment:
3 line file for 2 reasons.  First, it gives us a good place to put functions that are common (even though it isn't done yet), but it also gives us a central place to set the element max (and also set it manually through the regex method later!)